config/application.rb

```diff
@@ -18,7 +18,7 @@ class Application < Rails::Application
 
     config.middleware.insert_before 0, Rack::Cors do
       allow do
-        origins 'app.forestadmin.com', 'future.forestadmin.com'
```

Comment:
This is not a development repo, it is really related to the live demo, and doesn't have any relation with development environment